www/include/reporting/dashboard/xmlInformations/GetXmlHost.php

```diff
@@ -42,28 +42,30 @@
     exit;
 }
 
-if (isset($_GET["id"]) && isset($_GET["color"])) {
-    /* Validate the type of request arguments for security */
-    if (!is_numeric($_GET['id'])) {
-        $buffer->writeElement('error', 'Bad id format');
-        $buffer->endElement();
-        header('Content-Type: text/xml');
-        $buffer->output();
-        exit;
-    }
-
-    $color = array();
-    foreach ($_GET["color"] as $key => $value) {
-        $color[$key] = htmlentities($value, ENT_QUOTES, "UTF-8");
-    }
+$color = array_filter($_GET['color'] ?? false, function($oneColor) {
```

Comment:
if nothing is sent if the $_GET['color'], array_filter will throw a warning
as it expects an array, and a boolean is given as default value